src/XMakeBuildEngine/UnitTests/project.json

```diff
@@ -24,7 +26,8 @@
         "System.Threading.ThreadPool": "4.0.10",
         "System.Xml.XmlDocument": "4.0.1",
         "System.Xml.XPath.XmlDocument": "4.0.1",
-        "Microsoft.Win32.Primitives":  "4.0.1",
+        "Microsoft.Win32.Primitives": "4.0.1",
+        "Microsoft.Net.Compilers.NetCore": "2.0.0-beta3",
```

Comment:
I have no idea how this project was working before, without this reference . . .